### ISSUE TITLE
if width and height value is small ,the border will have a defect

### ISSUE THREAD

**Issue by Governhong1205** [Boris-Em/BEMCheckBox#4]
![734b6577-d782-4fec-b154-e48e7093a2c4](https://cloud.githubusercontent.com/assets/7943344/10656980/daca0c8c-78b7-11e5-961d-b145efc42b64.png)

**Comment by Boris-Em** [comment]
Thank you @Governhong1205.
The rotation seems to be causing this issue.
```objective-c
[path applyTransform:CGAffineTransformRotate(CGAffineTransformIdentity, M_PI / 2)];
```
I will investigate more and come up with a fix as soon as possible.

**Comment by A1exandre** [comment]
Same issue here
![screen shot 2015-11-02 at 12 50 45 pm](https://cloud.githubusercontent.com/assets/15021331/10889501/65a0859c-8160-11e5-9cb7-fcbed93e90b9.png)

**Comment by Governhong1205** [comment]
I tried to do like this , it will be ok
![image](https://cloud.githubusercontent.com/assets/7943344/10899478/fcaa9342-820f-11e5-869c-0ec520d4c7ca.png)

**Comment by A1exandre** [comment]
Worked perfectly !  Thanks

I have to do this code again every time I update my cocoapods though. If you have time to update your code with a fix it would be great ;)

![screen shot 2015-11-04 at 11 45 31 am](https://cloud.githubusercontent.com/assets/15021331/10944962/98ba5720-82e9-11e5-8b48-d90207e0eb67.png)

**Comment by Boris-Em** [comment]
This fix works, but breaks some of the animations unfortunately.
I need to find time to work on a "clean fix". Thank you your feedback!

**Comment by Governhong1205** [comment]
Are you talking about the OneStroke type? @Boris-Em 
I think using this way is temporarily fit to me because I dont use this type now.^_^.

**Comment by Boris-Em** [comment]
Yes, that's the one I'm talking about @Governhong1205. It's probably not the most popular one anyway.
So if you're not using it, you should be fine!
